portfolio.html

```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
 
     </section>
 
+    <section class="work">
```

Comment:
- [ ] like the previous reviewers requested , kindly add the styling for Active, Hover, and Pressed , for each button on your page 

![image](https://user-images.githubusercontent.com/89282221/185765590-fd0c00d3-29c0-4ce5-8e57-eceed020cfa5.png)